api/db/index.js

```diff
@@ -28,7 +28,14 @@ const setup = (
   models.forEach(modelObjects => {
     Object.keys(modelObjects).forEach(modelName => {
       logger.silly(`loading model [${modelName}]`);
-      const model = orm.Model.extend(modelObjects[modelName]);
+
+      const modelObj = modelObjects[modelName];
+      let staticProps = { modelName };
+      if (modelObj.static) {
+        staticProps = { modelName, ...modelObj.static };
+      }
+
+      const model = orm.Model.extend(modelObjects[modelName], staticProps);
```

Comment:
The second argument is for class properties.  Oddly, it will ignore a `name` property. 🤷‍♂️